fundingmanager_test.go

```diff
@@ -868,7 +900,16 @@ func TestFundingManagerNormalWorkflow(t *testing.T) {
 	fundingOutPoint := openChannel(t, alice, bob, 500000, 0, 1, updateChan,
 		true)
 
-	// Notify that transaction was mined
+	// Make sure both reservations time out and then run both zombie sweepers.
+	time.Sleep(1 * time.Millisecond)
+	alice.fundingMgr.pruneZombieReservations()
```

Comment:
Note that in case they in face **do** prune the reservations here, this will block, making the test timeout instead of fail. Instead you should do something similar as you did in the other tests, checking that an error is _not_ sent:

```go
go pruneAlice()
go pruneBob()

select {
case <-aliceMsgChan:
      t.Fatalf("error sent unexpectedly")
case <- time.After(100 * time.Millisecond): // note that we use a bit higher timeout, to make sure the go routine spins up
       // expected
}

...same for bob...
```

Reply:
Fixed.